### DISCUSSION TITLE
What else do you want new?

### DISCUSSION THREAD

**Discussion by januda-ui** [januda-ui/DRAGON-VPS-MANAGER#2] [Ideas]
👀

**Comment by Ahmad77888** [comment]
There should be an option to also limit users based on the traffic they use. A traffic limit along side with date limit is required.

**Comment by successAmebley** [comment]
can you add the banner script  to your ReadMe

**Comment by Eurovpns** [comment]
any body know how to temporarily disable a specific user?
